assets/app.js

```diff
@@ -760,6 +720,7 @@ new Vue({
                     'alias_ru': this.newResourceAliasDefault,
                     'sv': this.newResourceLabelDefault,
                     'alias_sv': this.newResourceAliasDefault,
+                    'translation_requested': '1',
```

Comment:
question: is this supposed to be a string 1? or can it be a numeric 1 or why not a boolean?

Reply:
yes, this should be of type `boolean: true/false`